keyboards/keychron/q2/config.h

```diff
@@ -103,4 +103,7 @@
 #define ENABLE_RGB_MATRIX_SPLASH
 // #define ENABLE_RGB_MATRIX_MULTISPLASH
 #define ENABLE_RGB_MATRIX_SOLID_SPLASH
-#define ENABLE_RGB_MATRIX_SOLID_MULTISPLASH
+// #define ENABLE_RGB_MATRIX_SOLID_MULTISPLASH
+
+#define RGB_MATRIX_KEYPRESSES
+#define RGB_MATRIX_FRAMEBUFFER_EFFECTS
```

Comment:
This was already addressed in the previous KC that was merged earlier.
* https://github.com/qmk/qmk_firmware/pull/19685/

Reply:
All right, correct now, thanks~